src/controlPSECount.h

```diff
@@ -11,6 +11,8 @@
 #include "uncrustify_types.h"
 
 
+void controlPSECountMinus(size_t value);
+
 void controlPSECount(size_t value);
```

Comment:
Why do we need both of those functions scattered throughout the `indent.cpp` file? The `frm.pse_tos` var is only changed in the pop and push functions. The checks should be done there. If we need more control than that we should make `frm` a class with a private `pse_tos`.

Reply:
No. The variable frm.pse_tos might get the value zero. And zero - 1 is "too big"